src/battle_pyramid_bag.c

```diff
@@ -622,7 +622,7 @@ static void CopyBagItemName(u8 *dst, u16 itemId)
 {
     if (ItemId_GetPocket(itemId) == POCKET_BERRIES)
     {
-        ConvertIntToDecimalStringN(gStringVar1, ITEM_TO_BERRY(itemId), STR_CONV_MODE_LEADING_ZEROS, 2);
+        ConvertIntToDecimalStringN(gStringVar1, ITEM_TO_BERRY(itemId), STR_CONV_MODE_LEADING_ZEROS, MAX_ITEM_DIGITS);
```

Comment:
This should use a `MAX_PYRAMID_ITEM_DIGITS` instead, since `MAX_ITEM_DIGITS` is for item bag.

Reply:
What should its value be? I didn't realize you were proposing an original change.

At the end of the day, the default limit of digits is 2, but it would be 3 if the based on `MAX_BAG_ITEM_CAPACITY`, so I don't see a reason not to use `MAX_ITEM_DIGITS` myself.

Reply:
Should I just `#define MAX_PYRAMID_ITEM_DIGITS   MAX_ITEM_DIGITS`?

I don't see much of a point, buut... 🤷

Reply:
`#define MAX_PYRAMID_ITEM_DIGITS ((MAX_PYRAMID_BAG_ITEM_CAPACITY > 99) ? 3 : 2)`